src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/provisioner/main.py

```diff
@@ -71,20 +101,12 @@ def create_or_update_account(org_session, account, adf_role_name, account_id=Non
                 ]
             )['Regions']
         ]
-        args = (
-            (account_id, region, role)
-            for region in all_regions
-        )
-        with ThreadPoolExecutor(max_workers=10) as executor:
-            for _ in executor.map(lambda f: schedule_delete_default_vpc(*f), args):
-                pass
-
-    LOGGER.info(f'Ensuring account alias for {account_id} of {account.alias}')
-    org_session.create_account_alias(account.alias, role)
-
-    if account.tags:
-        LOGGER.info(f'Ensuring tags exist for account {account_id}: {account.tags}')
-        org_session.create_account_tags(account_id, account.tags)
+        LOGGER.info(f'Regions are: {all_regions}')
+        return all_regions
+    except Exception as ce:
+        print(ce)
+        LOGGER.info(f'I am going to retry')
```

Comment:
can we update this info output to contain a bit of information and not use first person context :)